app/combine-statics/bin/combine-statics

```diff
@@ -40,6 +40,13 @@ for comp in $(ls); do
     mkdir -p $outputDir/$comp
     cdo -O merge *.nc $outputDir/$comp/$comp.static.nc
 
+#   #check
+#    if [ -z "$( ls $outputDir/$comp)" ]; then
+#        echo "empty"
+#    else
+#        echo "$(ls $outputDir/$comp)"
+#    fi
+
```

Comment:
Thanks so much Dana for this big update. It works well, hooray!

A commented debug code block here is a victimless crime in this shell script whose days are numbered...